### PR TITLE
feat: update api for spl-governance TVL

### PR DESCRIPTION
**Pull request by 0xShuk**
- This PR updates the TVL calculation for the tokens held by DAOs in the SPL Governance program. 
- The total TVL in usd value is updated at: https://realms-tvl.vercel.app/tvl/latest weekly on Sunday.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/spl-governance exports TVL: 
        
 
 
    solana                    927.41 M
    
    total                    927.41 M

File: projects/spl-governance/index.js [hunk 1]
```diff
@@ -1,22 +1,13 @@
 const { getCache } = require('../helper/http')
-const url = 'https://api.realms.today/stats/tvl'
+const url = 'https://realms-tvl.vercel.app/tvl/latest'
```
Comment (g1nt0ki) [inline projects/spl-governance/index.js:2]: thanks for the update, but we would like a token breakdown, or even better a way to pull the values on chain

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by llamatester** [comment]
The adapter at projects/spl-governance exports TVL: 
        
 
 
    solana                    1.28 B
    
    total                    1.28 B

File: projects/spl-governance/index.js (hunk 1 above)
Comment (0xShuk) [reply projects/spl-governance/index.js:2]: > thanks for the update, but we would like a token breakdown, or even better a way to pull the values on chain

The API has been updated to provide token-wise data. Fetching the token amounts for every DAO has a lot of overhead so we are providing it via API but we have included the treasuries holding each token in the API for additional verification if needed. Let me know if any further change is required, thanks.

**Comment by llamatester** [comment]
The adapter at projects/spl-governance exports TVL: 
        
 
 
    solana-staking            1.12 B
    staking                   1.12 B
    solana                    19.10 M
    
    total                    19.10 M

**Comment by llamatester** [comment]
The adapter at projects/spl-governance exports TVL: 
        
 
 
    solana-staking            1.17 B
    staking                   1.17 B
    solana                    42.25 M
    
    total                    42.25 M

**Comment by g1nt0ki** [comment]
sorry, we cant treat projects gov token as tvl, because they mint and deposit it, for example JITO: https://solscan.io/account/2Ch9AWnbAaummLkWTTtNgTAvrFq8YMATUaaN77TB2Y6C

**Comment by g1nt0ki** [comment]
would it be possible to get a list of projects, linked wallets and their gov token? we could list each individual project treasury

**Pull request merged**